hbase-replication/src/main/java/org/apache/hadoop/hbase/replication/FSReplicationPeerStorage.java

```diff
@@ -111,6 +113,7 @@ public void addPeer(String peerId, ReplicationPeerConfig peerConfig, boolean ena
       if (!enabled) {
         fs.createNewFile(new Path(peerDir, DISABLED_FILE));
       }
+      fs.createNewFile(new Path(peerDir, CREATE_TIME_FILE));
```

Comment:
I have changed it to just create this file and not add content. So I can use fs.getFileStatus(createTimeFile).getModificationTime() to get peer create  time. @Apache9